### PR TITLE
Style edits

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

File: _pages/index.md [hunk 1]
```diff
@@ -3,33 +3,31 @@ title: Introduction
...
 
 * **This list is:**
     * A general guide to review early and often
-    * _Especially_ important when you are starting to consider a future project launch or feature release
+    * _Especially_ important when you're starting to consider a future project launch or feature release
 * **This list is not:**
     * A last-minute pre-launch checklist, despite the "Before You Ship" title
 
-### Compliance
+## Compliance
```
Comment (afeld) [inline _pages/index.md]: Mind changing these back? The page title is an `<h2>`, so we want these to be lower.

File: _pages/index.md [hunk 2]
```diff
@@ -3,33 +3,31 @@ title: Introduction
...
 * You need to share details about your project and develop a communications plan with our outreach team (who will also help coordinate with the necessary GSA front offices, such as press and congressional affairs). To do this, complete this [communications plan template](https://docs.google.com/document/d/1xc7H6m7lfesCN-phJGvGSDPmtoinB5sM9KAA8deMNTQ/edit).
 * Share the good news about your launch with everyone at 18F by telling Kara DeFrias or Leah Gitter that you'd like to announce something at the next Tuesday all hands meeting.
 
-### Client expectations
+## Client expectations
 
-* Exactly how big of a splashy launch are you planning? Is POTUS announcing it? Have you decided on the level of spikes in traffic your budget supports? You should coordinate this with your engineers, client, and devops.
+* Exactly how big of a splashy launch are you planning? Is POTUS announcing it? Have you decided on the level of spikes in traffic your budget supports? You should coordinate this with your engineers, client, and DevOps.
```
Comment (afeld) [inline _pages/index.md]: Good catch! Mind changing it to Infrastructure? https://github.com/18F/DevOps/issues/532, if you'd like more background there.

**Comment by afeld** [comment]
Awesome, thank you! Just a couple small things.

**Comment by awfrancisco** [comment]
Made those changes.

**Pull request merged**